src/main/com/yetanalytics/lrs.cljc

```diff
@@ -21,7 +21,7 @@
   "Get information about this LRS"
   [lrs auth-identity]
   (try (p/-get-about lrs auth-identity)
-       (catch Exception ex
+       (catch #?(:clj Exception :cljs js/Error) ex
          {:error ex})))
```

Comment:
As mentioned in Slack, you should be able to use macros to avoid writing these try-catch blocks a million times.

Reply:
I'll try this, but not spending a lot of time on it if cljs becomes a problem

Reply:
https://github.com/yetanalytics/lrs/pull/71/commits/d940f9264455ef083e37e7f216a576debaded4d9